Confus/Confus/EventManager.cpp

```diff
@@ -0,0 +1,62 @@
+#include <irrlicht/irrlicht.h>
+
+#include "EventManager.h"
+
+namespace Confus
+{
+    struct SMouseState
```

Comment:
Is (probably unintended) global instead of a member variable, uncoventional type naming